example-get-started-experiments/generate.sh

```diff
@@ -67,56 +71,45 @@ tick
 git commit -m "Initialize DVC project"
 
 
-cp -r $HERE/code/data .
-git add data/.gitignore data/pool_data.dvc
+cp -r $HERE/code/datasets .
+git add datasets/.gitignore datasets/pool_data.dvc
 tick
 git commit -m "Add data"
 dvc pull
 
 
-cp -r $HERE/code/notebooks .
+cp -r $HERE/code/TrainSegModel.ipynb .
 git add .
 git commit -m "Add notebook using DVCLive"
 
+sudo apt-get update && sudo apt-get install ffmpeg libsm6 libxext6 -y
 pip install -r requirements.txt --extra-index-url https://download.pytorch.org/whl/cu118
 pip install jupyter
-jupyter nbconvert --execute 'notebooks/TrainSegModel.ipynb' --inplace
-# Apply best experiment
-BEST_EXP_ROW=$(dvc exp show --drop '.*' --keep 'Experiment|evaluate/dice_multi|base_lr' --csv --sort-by evaluate/dice_multi | tail -n 1)
-BEST_EXP_NAME=$(echo $BEST_EXP_ROW | cut -d, -f 1)
-BEST_EXP_BASE_LR=$(echo $BEST_EXP_ROW | cut -d, -f 3)
-dvc exp apply $BEST_EXP_NAME
+yolo settings datasets_dir="/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/datasets/"
+yolo settings runs_dir="/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/runs/"
+yolo settings weights_dir="/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/weights/"
```

Comment:
Why is it needed in this script but doesn't seem to be required when running in the published repo?

Reply:
Because in the codespace we are running `bash generate.sh` and inside that we are changing the working directory which messes with where YOLO is looking for the dataset